packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -76,33 +76,58 @@ enum NetworkType {
 export function SwitchNetworksButton(
   props: React.ButtonHTMLAttributes<HTMLButtonElement>
 ) {
-  const {
-    isEOA,
-    isSmartContractWallet,
-    isLoading: isLoadingAccountType
-  } = useAccountType()
+  const { isSmartContractWallet, isLoading: isLoadingAccountType } =
+    useAccountType()
 
   return (
     <button
       type="button"
       disabled={isSmartContractWallet || isLoadingAccountType}
       className={twMerge(
-        'arb-hover flex h-12 w-12 items-center justify-center rounded-full bg-white p-2 shadow-[0_0_4px_0_rgba(0,0,0,0.25)] transition duration-200',
-        isEOA
-          ? 'hover:animate-rotate-180 focus-visible:animate-rotate-180 hover:bg-[#F4F4F4] focus-visible:ring-2 focus-visible:ring-gray-4 active:bg-gray-2'
-          : ''
+        'group flex h-7 w-7 items-center justify-center rounded bg-[#191919] p-1',
+        (isSmartContractWallet || isLoadingAccountType) && 'pointer-events-none'
       )}
       {...props}
     >
       {isSmartContractWallet ? (
-        <ChevronDownIcon className="h-6 w-6 stroke-2 text-dark" />
+        <ArrowDownIcon className="h-6 w-6 stroke-2 text-white" />
       ) : (
-        <ArrowsUpDownIcon className="h-6 w-6 stroke-2 text-dark" />
+        <ArrowsUpDownIcon className="h-8 w-8 stroke-2 text-white transition duration-300 group-hover:rotate-180 group-hover:opacity-80" />
```

Comment:
stroke-1 looks better now as it's quite small
stroke-2 is very thick for this size